src/BenchmarkDotNet/ConsoleArguments/CommandLineOptions.cs

```diff
@@ -187,6 +187,9 @@ public class CommandLineOptions
         [Option("AOTCompilerMode", Required = false, Default = MonoAotCompilerMode.mini, HelpText = "Mono AOT compiler mode, either 'mini' or 'llvm'")]
         public MonoAotCompilerMode AOTCompilerMode { get; set; }
 
+        [Option("wasmDataDir", Required = false, HelpText = "Wasm data directory")]
+        public DirectoryInfo WasmDataDirectory { get; set; }
```

Comment:
are there any scenarios in which this argument is mandatory? If so the following method should take that into account:

https://github.com/dotnet/BenchmarkDotNet/blob/5852a1340653250f1fa8801102556026d8f87e45/src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs#L100

Reply:
It's not mandatory.